yamls/jp/overlays.yaml

```diff
@@ -17312,7 +17312,7 @@
     bss_size: 0x0
     subsegments:
       - [0x970880, c, overlays/actors/ovl_Dummy/ac_dumm]
-      - [0x9708C0, data, overlays/actors/ovl_Dummy/ac_dumm]
+      - [0x9708C0, .data, overlays/actors/ovl_Dummy/ac_dumm]
```

Comment:
Can you rename this definition to ac_dummy, and ac_dumm.c to ac_dummy.c? I think this is a typo from when we first generated the actor stuff.

Reply:
I can confirm, it is a typo.
I made those by hand I guess I messed up this one